### PR TITLE
feat(coalesce-same-type-notifications): Coalesce same type notificati…

### PR DESCRIPTION
**Pull request by tudor-phd**
…ons.

### CODE REVIEW

**Comment by gpatel-fr** [comment]
Nice feature, spelling in 'raisedHandLenght' though :-)

File: react/features/base/participants/constants.js [hunk 1]
```diff
@@ -60,6 +60,13 @@ export const PARTICIPANT_JOINED_SOUND_ID = 'PARTICIPANT_JOINED_SOUND';
  */
 export const PARTICIPANT_LEFT_SOUND_ID = 'PARTICIPANT_LEFT_SOUND';
 
+/**
+ * The identifier of the raise hand notification.
+ *
+ * @type {string}
+ */
+export const RAISE_HAND_NOTIFICATION_ID = 'RAISE_HAND_NOTIFICATION';
```
Comment (saghul) [inline react/features/base/participants/constants.js:68]: Can we have this constant in the notifications feature please?

**Review by saghul — COMMENTED** [review]
One minor nit, LGTM otherwise, I'll give it a test!

File: react/features/notifications/constants.js [hunk 2]
```diff
@@ -50,3 +50,15 @@ export const NOTIFICATION_TYPE_PRIORITIES = {
...
+ * Amount of participants beyond which no left notification will be emitted.
+ */
+export const SILENT_LEFT_THRESHOLD = 30;
+
+/**
+ * The identifier of the raise hand notification.
+ *
+ * @type {string}
+ */
+export const RAISE_HAND_NOTIFICATION_ID = 'RAISE_HAND_NOTIFICATION';
```
Comment (saghul) [inline react/features/notifications/constants.js:64]: sort alphabetically

**Review by saghul — APPROVED** [review]

**Pull request merged**